README.md

```diff
@@ -102,10 +102,6 @@ You can play with the PDF.js API directly from your browser using the live demos
 
 + [Interactive examples](http://mozilla.github.io/pdf.js/examples/index.html#interactive-examples)
 
-The repository contains a hello world example that you can run locally:
-
-+ [examples/helloworld/](https://github.com/mozilla/pdf.js/blob/master/examples/helloworld/)
-
 More examples can be found in the examples folder. Some of them are using the pdfjs-dist package, which can be built and installed in this repo directory via `gulp dist-install` command.
```

Comment:
Nit: Maybe it wouldn't be a bad idea to change the "examples folder" text into a link, i.e. like so:
[examples folder](https://github.com/mozilla/pdf.js/tree/master/examples/)
```